src/lib/containers/widgets/query-filter/FacetFilterControls.tsx

```diff
@@ -137,19 +137,23 @@ export const applyChangesToRangeColumn = (
   onChangeFn(result)
 }
 
-export const QueryFilter: React.FunctionComponent<QueryFilterProps> = ({
-  facetsToFilter,
-}): JSX.Element => {
+export function FacetFilterControls(props: FacetFilterControlsProps) {
+  const { facetFiltersToShow } = props
   const { data, isLoadingNewBundle, getLastQueryRequest, executeQueryRequest } =
     useQueryContext()
 
-  const facets = data?.facets
+  const facets = data?.facets?.filter(
+    facet =>
+      // Don't show facets where there are no values
+      !isSingleNotSetValue(facet),
+  )
```

Comment:
Here's the change. The filter was previously applied just on the control, so the chip would render but the control wouldn't be shown when the chip was clicked.